encoder_options.go

```diff
@@ -0,0 +1,34 @@
+package enmime
+
+type EncoderOption interface {
+	apply(p *Encoder)
+}
+
+// Encoder implements MIME part encoding options
+type Encoder struct {
+	// Enforce Quoted encoding when selecting Content Transfer Encoding
+	enforceQuotedCte bool
```

Comment:
In my opinion, `enforce` is the wrong term, as quoted-cte is not a rule or policy.  Just use `force` here and elsewhere.